core/error/error_macros.h

```diff
@@ -487,34 +515,37 @@ void _err_flush_stdout();
  * Ensures `m_cond` is false.
  * If `m_cond` is true, the current loop continues.
  */
-#define ERR_CONTINUE(m_cond)                                                                                       \
-	if (unlikely(m_cond)) {                                                                                        \
-		_err_print_error(FUNCTION_STR, __FILE__, __LINE__, "Condition \"" _STR(m_cond) "\" is true. Continuing."); \
-		continue;                                                                                                  \
-	} else                                                                                                         \
-		((void)0)
+#define ERR_CONTINUE(m_cond)                                                                                           \
+	do {                                                                                                               \
+		if (unlikely(m_cond)) {                                                                                        \
+			_err_print_error(FUNCTION_STR, __FILE__, __LINE__, "Condition \"" _STR(m_cond) "\" is true. Continuing."); \
+			continue;                                                                                                  \
```

Comment:
This changes how `continue` work here.

Reply:
Yeah, the break/continue macros are fine as is so I have reverted those.

Reply:
Seems like you only reverted one of the three, and it's now broken.

Reply:
Apparently I forgot to add my latest edits.  The intent was all the break/continue version stay as they are now using ((void)0) since we don't want to change the behavior of the macros used inside loops.  Macros that only return or generate a trap can be wrapped.  I'll rebase and push again after a clean build.